main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"context"
+	"os"
+	"strconv"
+
+	"github.com/ONSdigital/dp-reporter-client/reporter"
+	"github.com/ONSdigital/dp-search-builder/config"
+	"github.com/ONSdigital/dp-search-builder/elasticsearch"
+	"github.com/ONSdigital/dp-search-builder/service"
+	"github.com/ONSdigital/go-ns/kafka"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/go-ns/rchttp"
+)
+
+func main() {
+	log.Namespace = "dp-search-builder"
+
+	cfg, err := config.Get()
```

Comment:
Consider logging out config values on start of the app. Be sure to not log any sensitive values

Reply:
Done